proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/configuration/TranslatorConfiguration.java

```diff
@@ -69,11 +48,7 @@ public String getConnectorNodeIssuerId() {
         return connectorNodeIssuerId;
     }
 
-    public String getKeyRetrieverServiceName() { return keyRetrieverServiceName; }
-
-    public String getSoftHSMLibPath() { return softHSMLibPath; }
-
-    public String getSoftHSMSigningKeyPin() { return softHSMSigningKeyPin; }
```

Comment:
Can the rest of the Soft HSM be removed now? I.E HelmCharts, dockerfile etc. Not in this ticket but just generally.

Reply:
Yeah I've already done some of the cleanup in the Dockerfile and will do the Helm chart stuff in a separate PR.